README.md

```diff
@@ -1,35 +1,54 @@
-# status: Pre-Alpha
+# Current version: Pre-Alpha
 
-# osu-radio
+Roadmap | Alpha Features & Issues: https://github.com/orgs/Team-BTMC/projects/2
+<br>
+Main Figma Design: [https://www.figma.com/design/tNBJr7TlEsoWsWdAewqoUg](https://www.figma.com/design/tNBJr7TlEsoWsWdAewqoUg/osu!-radio?node-id=0-1&t=aIuThZAj00HcSjM5-1)
 
-Application to play your osu! songs.
+[//]: # (You can remove the ${\textsf{\color{#f462a3}osu!}}$ if you just want it be osu!radio instead. That color can only work if there's spaces on both side)
 
-![image](https://github.com/user-attachments/assets/0cdfd7a0-3e08-4c5b-bb11-423c74176c74)
+# What is ${\textsf{\color{#f462a3}osu!}}$ radio
+It's an application that play your ${\textsf{\color{#f462a3}osu!}}$ songs!
 
-This is primarily a personal project so there could be "idk, works for me" and if it is not a trivial fix I might not ever fix it. Also, if it breaks then... RIPBOZO
+![Song Tab](https://github.com/user-attachments/assets/da67b906-1429-4cc1-9087-76026e94b98a "The screen show a UI with all the buttons (Play, Pause, Forward and Rewind, as well as a Seek bar, with 4 songs on the left")
 
-![](https://cdn.7tv.app/emote/60dbea82e3e5887a4a95e1db/4x.webp)
+### Download Guide
+> [!NOTE]
+> To use `git` and `npm`, you need to have [git](https://git-scm.com/) & [node.js](https://nodejs.org/en) installed
 
-### Contribution
-
-I only know how to push code from my local machine to this repository, so, you can forget about pull requests LOL. Definitely a skill issue.
-
-### Download
+`1` | Open the terminal in the directory you want to download osu!radio in. Then do the following command,
+```
+git clone https://github.com/Team-BTMC/osu-radio.git
+```
+> This command will create a copy of the github repositories in that directory
 
-LOL, it can barely play songs. You better wait a few months. But if you still want to give it a try, then git clone this repository and run:
+`2` | Then move into that directory with the following command,
+```
+cd .\osu-radio\
+```
 
+`3` | Next, do the following command to install the necessary files to run osu!radio
 ```
 npm install
-npm run start
 ```
 
-# THE VISION
+`4` | To start it up, simply fun the following command inside that osu!radio directory
+```
+npm run start
+```
 
-Technologies:
-- Electron
-- Solid js
-- Typescript
+### Contribution
+You can create a fork of the [main repositories](https://github.com/Team-BTMC/osu-radio) and do PR from there
```

Comment:
Should include more decent explaination of best habits for when making a pull request, deciding stuff, etc. @CaptSiro, can you write one or should I help with workflow like this?